python-sdk/tutorials/deploy-with-prebuilt-docker-images/pytorch-prebuilt-docker-image.ipynb

```diff
@@ -0,0 +1,938 @@
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.048629,
+          "end_time": "2021-05-18T20:36:50.604201",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:50.555572",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Write the requirements.txt\n",
+        "Write the `requirements.txt`\n",
+        "\n",
+        "> **Pin each package version:**\n",
+        "Even while prototyping, pin each package version in requirements.txt. \n",
+        "For example, use scipy == 1.2.3 instead of just scipy or even scipy > 1.2.3. \n",
+        "If you don't pin an exact version and scipy releases a new version, this can break your scoring script and cause failures during deployment and scaling."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.051314,
+          "end_time": "2021-05-18T20:36:50.712818",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:50.661504",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%writefile requirements.txt\n",
+        "  torchvision==0.7.0\n",
+        "  future==0.17.1\n",
+        "  pillow==8.1.1\n",
+        "  azureml.core==1.25.0"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "collapsed": true,
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.158794,
+          "end_time": "2021-05-18T20:36:50.920819",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:50.762025",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Define the environment (Dynamic installation via requirements.txt)\n",
+        "\n",
+        "Then, we will need to create an Azure Machine Learning environment that specifies all of the scoring script's package dependencies. \n",
+        "Here we assume the `requirements.txt` is in the right format and inlcudes all additional packages for the model.\n",
+        "\n",
+        "If deployment fails because of Python extensibility solution, there will be no logs available the [Azure Machine Learning Studio](https://ml.azure.com),`service.get_logs()` will return None. If there is a problem in the *init()* function of score.py, `service.get_logs()` will return logs for the same.\n",
```

Comment:
lowercase "studio", "Studio" refers to the sunset Studio classic product

Reply:
done